hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -95,13 +96,19 @@ const Orders = () => {
                                             history.push(`/teams/${order.team_code}`)
                                         }
                                     >
-                                        <OrderCard
-                                            teamCode={order.team_code}
-                                            orderQuantity={order.items.length}
-                                            time={order.created_at}
-                                            id={order.id}
-                                            status={order.status}
-                                        />
+                                        {(order.status === "Submitted" ||
+                                            order.status === "Picked Up" ||
```

Comment:
Why we are doing this? To avoid the corner cases right? I think this might be a bit of hardcoding. Are there any better way to do this Ethan? @YuYing-Liang @Mustaballer   any comment on this?

Reply:
This is one way of writing it
![image](https://user-images.githubusercontent.com/43456930/211226029-ae490146-568b-4a09-b1c4-a0232288fe42.png)
If you want I can declare a global array of the statuses, and substitute it in the image @Leo6Leo

Reply:
The one showed in your screenshot looks good to me!